services/netlify/netlify.service.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const { renderBuildStatusBadge } = require('../build-status')
+const { BaseSvgScrapingService } = require('..')
+
+const pendingStatus = 'building'
+const notBuiltStatus = 'not built'
+
+const statusMap = {
+  building: pendingStatus,
+  waiting: pendingStatus,
+  initiated: pendingStatus,
+  stopped: notBuiltStatus,
+  ignored: notBuiltStatus,
+  blocked: notBuiltStatus,
+  infrastructure_failure: 'failed',
+}
+
+module.exports = class Netlify extends BaseSvgScrapingService {
+  static get category() {
+    return 'build'
+  }
+
+  static get route() {
+    return {
+      base: 'netlify',
+      pattern: ':projectId',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Netlify',
+        namedParams: {
+          projectId: 'e6d5a4e0-dee1-4261-833e-2f47f509c68f',
+        },
+        staticPreview: renderBuildStatusBadge({ status: 'passing' }),
+      },
+      // {
+      //   title: 'Netlify (branch)',
+      //   pattern: ':projectId/:branch',
+      //   namedParams: {
+      //     projectId: '0bdb0440-3af5-0133-00ea-0ebda3a33bf6',
+      //     branch: 'master',
+      //   },
+      //   staticPreview: renderBuildStatusBadge({ status: 'passing' }),
+      // },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'netlify',
+      namedLogo: 'netlify',
+    }
+  }
+
+  static render({ status }) {
+    status = statusMap[status] || status
+    let result = renderBuildStatusBadge({ status })
```

Comment:
The linter requires this to be `const`.

```suggestion
    const result = renderBuildStatusBadge({ status })
```